<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommunicationBoard</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Resources/icon.png</normaloff>../Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="CategoryName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>1121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>הכנס את שם הקטגוריה</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>320</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>הוספת קטגוריה</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AddCategoryButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>800</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>הוסף קטגוריה</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Resources/UI Icons/plus-circle.svg</normaloff>../Resources/UI Icons/plus-circle.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorMsgLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>850</y>
      <width>520</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>שגיאה</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>בחר תמונה לקטגוריה:</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectedIcon">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
